packages/clay-management-toolbar/demos/index.html

```diff
@@ -351,6 +351,48 @@ <h2>With results bar and filter labels</h2>
 		},
 		'#active-checked-select-all-block');
 
+		//Active state with checkbox checked selecting one item
+		new metal.ClayManagementToolbar({
+			actionItems: actionItems,
+			checkboxStatus: 'checked',
+			selectable: true,
+			selectedItems: 1,
+			showInfoButton: true,
+			showSelectAllButton: true,
+			spritemap: spritemap,
+			supportsBulkActions: true,
+			totalItems: 10,
+		},
+		'#active-checked-select-all-block');
```

Comment:
Can you separate these new examples in blocks? the same way we do with existing ones.

Reply:
Yes!